src/StreamFactory/DiactorosStreamFactory.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Http Discovery package.
+ *
+ * (c) PHP HTTP Team <team@php-http.org>
+ *
+ * For the full copyright and license information, please read the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Http\Discovery\StreamFactory;
+
+use Http\Message\StreamFactory;
+use Psr\Http\Message\StreamInterface;
+use Zend\Diactoros\Request;
+use Zend\Diactoros\Response;
+use Zend\Diactoros\Stream;
+
+/**
+ * Zend Diactoros StreamFactory
+ *
+ * @author GeLo <geloen.eric@gmail.com>
```

Comment:
Actually, it is written by me. Back then I added Eric to each files he had any connection with, but that class is 90% rewritten by me. Even if it is copied partly, please add only yourself.

You might also add a comment that the code is copied from message factory. Actually, the ultimate solution would be somehow share the code between the two classes. But let's think about it later.